config/locales/en.yml

```diff
@@ -101,3 +101,129 @@ en:
+          - text: Use the ‘starting a business’ work as a model for simplifying other complex tasks
+      - heading: Users need content to be easy to find and understand, fully accessible, and accurate, both now and in the future
+        paragraph: Ensure that government content is easy to publish and find everywhere, and meets user needs
+        steps:
+        - heading: Recently shipped
+          items:
+          - text: Major updates to immigration guidance due to visa simplification work
+          - text: Make GOV.UK more accessible by making content and design elements compliant with new accessibility regulations
+          - text: Run a discovery project on the navigation systems used on GOV.UK
+          - text: Continuous updates and improvements to the most used content
+        - heading: Up next
+          items:
+          - text: Large scale changes to all guidance affected by Brexit
+          - text: Alpha designs for new GOV.UK navigation system
+          - text: Updating all the content affected by the rate changes ready for the 2021 to 2022 financial year
+        - heading: Exploring
+          items:
+          - text: Find efficiencies in how we manage the most used content
+          - text: Investigate what improvements can be made to the way we group content by what the content is about
+          - text: Run a discovery on the future development of GOV.UK publishing applications
+      - heading: We can use more of the data available to us to improve GOV.UK and online services
+        paragraph: Use data science, data analytics and research techniques to make services better and more efficient
+        steps:
+        - heading: Recently shipped
+          items:
+          - text: Create draft reports to see how services are being used, end-to-end
+          - text: Ongoing data analysis to help improve content and services related to coronavirus and Brexit
+        - heading: Up next
+          items:
+          - text: Build data visualisation tools so it’s easier to analyse how anonymous users interact with digital government services
+          - text: Automate the reports that show insights from coronavirus and Brexit content
+        - heading: Exploring
+          items:
+          - text: How to expand and automate data reports
+          - text: Further ways to automate analysis of user feedback so we can identify trends more quickly
+      - heading: Everyone in the UK needs to understand how they are affected by coronavirus and Brexit
+        paragraph: Create tools and guidance that help users find critical and up-to-date information
+        steps:
+        - heading: Recently shipped
+          items:
+          - text: Continuous updates to the coronavirus landing page and tools to help users find guidance, services and goverment support for coronavirus
+          - text: Launch a service for the public to ask ministers questions about coronavirus (gov.uk/ask)
```

Comment:
Should this be a link? It probably isn't in the design, but it looks odd to me as text.

Reply:
Good shout, I asked Keith and he agreed it should be a link.